src/main/java/net/atos/zac/app/bag/converter/RESTNummeraanduidingConverter.java

```diff
@@ -20,22 +23,31 @@
 
 public class RESTNummeraanduidingConverter {
 
+    @Inject
+    private RESTWoonplaatsConverter woonplaatsConverter;
+
+    @Inject
+    private RESTOpenbareRuimteConverter openbareRuimteConverter;
+
     public RESTNummeraanduiding convertToREST(final NummeraanduidingIOHalBasis nummeraanduidingIO) {
         if (nummeraanduidingIO == null) {
             return null;
         }
-        final Nummeraanduiding nummeraanduiding = nummeraanduidingIO.getNummeraanduiding();
-        final RESTNummeraanduiding restNummeraanduiding = new RESTNummeraanduiding();
+        final RESTNummeraanduiding restNummeraanduiding = convertToREST(nummeraanduidingIO.getNummeraanduiding());
         restNummeraanduiding.url = URI.create(nummeraanduidingIO.getLinks().getSelf().getHref());
-        restNummeraanduiding.identificatie = nummeraanduiding.getIdentificatie();
-        restNummeraanduiding.postcode = nummeraanduiding.getPostcode();
-        restNummeraanduiding.huisnummer = nummeraanduiding.getHuisnummer();
-        restNummeraanduiding.huisletter = nummeraanduiding.getHuisletter();
-        restNummeraanduiding.huisnummertoevoeging = nummeraanduiding.getHuisnummertoevoeging();
-        restNummeraanduiding.huisnummerWeergave = convertHuisnummerWeergave(nummeraanduiding);
-        restNummeraanduiding.status = nummeraanduiding.getStatus();
-        restNummeraanduiding.typeAdresseerbaarObject = nummeraanduiding.getTypeAdresseerbaarObject();
-        restNummeraanduiding.geconstateerd = Indicatie.J.equals(nummeraanduiding.getGeconstateerd());
+        return restNummeraanduiding;
+    }
+
+    public RESTNummeraanduiding convertToREST(final NummeraanduidingIOHal nummeraanduidingIO) {
+        if (nummeraanduidingIO == null) {
```

Comment:
Het is niet consistent dat converters de ene keer null safe zijn en de andere keer weer niet. Volgens mij zijn ze in principe nooit null safe,

Reply:
alle embedded kunnen null zijn, paar checks toegevoegd